core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -845,6 +845,7 @@ private void newTabInBackground(String url) {
   }
 
   private void closeTab(int index) {
+    currentWebViewIndex = 0;
```

Comment:
we should decrement the current index if the closing index is less than or equal and do nothing if greater than I would say

Reply:
Sounds better!